cf/mixin/propertiesdatabounds.py

```diff
@@ -3758,6 +3758,71 @@ def persist(self, bounds=True, inplace=False):
             inplace=inplace,
         )
 
+    @_inplace_enabled(default=False)
+    def rechunk(
+        self,
+        chunks=_DEFAULT_CHUNKS,
+        threshold=None,
+        block_size_limit=None,
+        balance=False,
+        bounds=True,
+        interior_ring=True,
+        inplace=False,
+    ):
+        """Change the chunk structure of the data.
+
+        .. versionadded:: TODODASKVER
+
+        .. seealso:: `cf.Data.rechunk`
+
+        :Parameters:
+
+            {{chunks: `int`, `tuple`, `dict` or `str`, optional}}
+
+            {{threshold: `int`, optional}}
+
+            {{block_size_limit: `int`, optional}}
+
+            {{balance: `bool`, optional}}
+
+            bounds: `bool`, optional
+                If False then do not rechunk any bounds. By default
+                any bounds are also rechunked.
+
+            interior_ring: `bool`, optional
+                If False then do not rechunk an interior ring array,
+                if one exists. By default an interior ring array is
+                rechunked.
```

Comment:
For the same reasons as my previous comment:

```suggestion
                If True (the default) rechunk an interior ring array,
                if one exists.
```